src/modules/tools/zprobe/CartGridStrategy.cpp

```diff
@@ -349,14 +424,14 @@ bool CartGridStrategy::handleGcode(Gcode *gcode)
 
 // These are convenience defines to keep the code as close to the original as possible it also saves memory and flash
 // set the rectangle in which to probe
-#define LEFT_PROBE_BED_POSITION (0)
+#define LEFT_PROBE_BED_POSITION (x_start)
 #define RIGHT_PROBE_BED_POSITION (x_size)
 #define BACK_PROBE_BED_POSITION (y_size)
```

Comment:
There two changes:
First i remove legacy from DeltaGrid - where left/rigth and front/back calculated from radius or width/length of table.
Then i add size changeable on the fly.
I think that this define can be removed. And may be chant to "this->" when using default values.